nand.py

```diff
@@ -1,27 +1,30 @@
 import os
 
+DEFAULT_DATA = "0x00000000"
+
 
 class Nand:
     def __init__(self, lba_length, data_file):
         self.data_file = data_file
         self.lba_length = lba_length
-        self.default_value = "0x00000000"
+        self.default_data = DEFAULT_DATA
 
     def create_data_file(self):
         with open(self.data_file, 'a', encoding='utf-8') as f:
-            for i in range(self.lba_length):
-                f.write(f"{self.default_value}\n")
+            for index in range(self.lba_length):
+                f.write(f"{self.default_data}\n")
 
     def read(self, address) -> str:
         if not os.path.exists(self.data_file):
             self.create_data_file()
 
         with open(self.data_file, 'r', encoding='utf-8') as f:
-            for i, line in enumerate(f, start=0):
-                if i == address:
-                    value = line.strip()
-                    return value
-        raise Exception
+            lines = f.readlines()
+
+        if address > len(lines):
```

Comment:
```suggestion
        if address >= len(lines):
```
이거 혹시 등호가 붙어야하는거 아닌가 생각이 듭니다.

예를들어 len(lines)==1이고 address == 1이면
27line에 lines[address] 가 index out of range 가 발생할것같습니다.